templates.json

```diff
@@ -0,0 +1,149 @@
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "node",
+        "source": "openfaas",
+        "description": "Official NodeJS 8 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "php7",
+        "source": "openfaas",
+        "description": "Official PHP 7 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python-armhf",
+        "source": "openfaas",
+        "description": "Official Python 2 armhf template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python",
+        "source": "openfaas",
+        "description": "Official Python 2 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python3-armhf",
+        "source": "openfaas",
+        "description": "Official Python 3 armhf template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "python3",
+        "source": "openfaas",
+        "description": "Official Python 3 x86-64 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "ruby",
+        "source": "openfaas",
+        "description": "Official Ruby x86-64 template",
+        "repo": "https://github.com/openfaas/templates",
+        "official": "true"
+    },
+    {
+        "name": "node10-express-template",
+        "source": "openfaas-incubator",
+        "description": "NodeJS 10 Express template",
+        "repo": "https://github.com/openfaas-incubator/node10-express-template",
+        "official": "false"
+    },
+    {
+        "name": "powershell-http-template",
+        "source": "openfaas-incubator",
+        "description": "PowerShell HTTP template",
+        "repo": "https://github.com/openfaas-incubator/powershell-http-template",
+        "official": "false"
+    },
+    {
+        "name": "ruby-http",
+        "source": "openfaas-incubator",
+        "description": "Ruby HTTP template",
+        "repo": "https://github.com/openfaas-incubator/ruby-http",
+        "official": "false"
+    },
+    {
+        "name": "python-flask-template",
+        "source": "openfaas-incubator",
+        "description": "Python 2.7/3.6 Flask template",
+        "repo": "https://github.com/openfaas-incubator/python-flask-template",
+        "official": "false"
+    },
+    {
+        "name": "node8-express-template",
+        "source": "openfaas-incubator",
+        "description": "NodeJS 8 Express template",
+        "repo": "https://github.com/openfaas-incubator/node8-express-template",
+        "official": "false"
+    },
+    {
+        "name": "golang-http-template",
```

Comment:
There's two templates in this repo btw.

Reply:
I know, do you mean I should add some array field which shows all templates inside this repository?

Reply:
This is now resolved thanks for the feedback on that one did not though of it in that way 🍻